flake.nix

```diff
@@ -153,11 +156,39 @@
           }];
 
         };
+
+        moduleTest = (nixpkgs.lib.nixos.runTest {
+          hostPkgs = pkgs;
+          defaults.documentation.enable = false;
+          imports = [{
+            name = "teslamate";
+            nodes.server = {
+              imports = [ self.nixosModules.default ];
+              services.teslamate = {
+                enable = true;
+                secrestFile = builtins.toFile "teslamate.env" ''
```

Comment:
Possible typo in `secretsFile` here.

Reply:
Oops yeah. Those force pushes earlier were fixing that, I guess I missed this one lol